.github/workflows/ci.yml

```diff
@@ -21,3 +21,6 @@ jobs:
 
     - name: Run all checks
       run: make all
+
+    - name: Run tests with race detection
+      run: go test -race ./...
```

Comment:
The previous step (`make all`) already runs tests.
Shouldn't we instead add `-race` there?
https://github.com/docker/model-distribution/blob/main/Makefile#L31